src/sharing/components/ShareByEmail.jsx

```diff
@@ -298,4 +366,6 @@ ShareByEmail.defaultProps = {
   contacts: []
 }
 
-export default ShareByEmail
+export default connect(null, dispatch => ({
+  renewAuthorization: client => dispatch(renewAuthorization(client))
+}))(ShareByEmail)
```

Comment:
Bah, I see why it's there but i's annoying to link this part of the code with the general redux store :(